packages/manager/apps/pci-quota/src/pages/quota/__snapshots__/Quota.page.spec.tsx.snap

```diff
@@ -0,0 +1,9 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`QuotaPage > should render 1`] = `
+<div>
```

Comment:
since this snapshot only displays a spinner, could you please mock the data hook so we can check that the page is renderered correctly